source/intro.md

```diff
@@ -245,7 +245,7 @@ is included with [the code for this book](https://github.com/UBC-DSCI/introducti
 If we were to open this data in a plain text editor (a program like Notepad that just shows
 text with no formatting), we would see each row on its own line, and each entry in the table separated by a comma:
 
-```code
+```text
```

Comment:
Let's leave this as-is and do a new PR that fixes this throughout the book

we should also check that changing from `code` to `text` doesn't change formatting for the worse (and if it does, we should find the "right" language tag to use)

Reply:
I'll open a PR for that now -- I'll just `sed` it

Reply:
see #189